### PR TITLE
Windows management from the render process

### PR DESCRIPTION
**Pull request by chicoxyzzy**
https://github.com/socketsupply/socket/pull/31

### CODE REVIEW

File: runtime.js [hunk 1]
```diff
@@ -161,11 +172,11 @@ export async function setContextMenu (o) {
     .entries(o)
     .flatMap(a => a.join(':'))
     .join('_')
-  return await ipc.send('context', o)
+  return ipc.send('context', o)
 }
```
Comment (jwerle) [inline runtime.js:148]: Why drop all the `await`?

Comment (chicoxyzzy) [reply runtime.js:148]: It's not necessary and can take an extra tick in old versions of WebKit

Comment (jwerle) [reply runtime.js:148]: Absolutely necessary when you want to preserve the call stack

see https://gist.github.com/jwerle/1bc93e303c3532abe0bdcbbea75e528d

Comment (chicoxyzzy) [reply runtime.js:148]: added again. BTW is this V8-only feature?

Comment (jwerle) [reply runtime.js:148]: Hmm actually it looks like it. Stack traces are garbage in WebKit

**Review by jwerle — APPROVED** [review]

**Pull request merged**